### PR TITLE
Add basic Dependabot config

### PR DESCRIPTION
**Pull request by MarcDenman**
This PR adds a basic Dependabot config to allow each repo to be picked up by Dependabot.

### CODE REVIEW

File: {{cookiecutter.project_slug}}/.dependabot/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+version: 1
+update_configs:
+  - package_manager: "python"
+    directory: "/"
+    update_schedule: "live"
```
Comment (eganjs) [inline {{cookiecutter.project_slug}}/.dependabot/config.yml]: Add a new line

**Comment by eganjs** [comment]
Should also have a `.dependabot/config.yml ` at the top level, for the project containing the template
Also I would like to see https://github.com/UKHO/gsfpy/blob/master/.github/workflows/auto-approve.yml in here as well, for the project and the template

**Comment by MarcDenman** [comment]
> Should also have a `.dependabot/config.yml ` at the top level, for the project containing the template
> Also I would like to see https://github.com/UKHO/gsfpy/blob/master/.github/workflows/auto-approve.yml in here as well, for the project and the template

Will add the autoapprove for the template. The stuff at the top level for this project should be a seperate PR.

**Comment by eganjs** [comment]
> Will add the autoapprove for the template. The stuff at the top level for this project should be a seperate PR.

I can agree to that

File: {{cookiecutter.project_slug}}/.dependabot/config.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
```
Comment (eganjs) [inline {{cookiecutter.project_slug}}/.dependabot/config.yml]: fix these line endings

Comment (MarcDenman) [reply {{cookiecutter.project_slug}}/.dependabot/config.yml]: How do you want them fixed?

Comment (eganjs) [reply {{cookiecutter.project_slug}}/.dependabot/config.yml]: Remove trailing lines and make existing lines match this specification: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

**Pull request merged**